docs/reference/release-notes/7.0.0-alpha2.asciidoc

```diff
@@ -1,10 +1,599 @@
+* Deprecate types in update_by_query and delete_by_query {pull}36365[#36365] (issue: {issue}35190[#35190])
+* For msearch templates, make sure to use the right name for deprecation logging. {pull}36344[#36344]
+* Deprecate types in termvector and mtermvector requests. {pull}36182[#36182]
+* Deprecate types in update requests. {pull}36181[#36181]
+* Deprecate types in document delete requests. {pull}36087[#36087]
+* Deprecate types in get, exists, and multi get. {pull}35930[#35930]
+* Deprecate types in search and multi search templates. {pull}35669[#35669]
+* Deprecate types in explain requests. {pull}35611[#35611]
+* Deprecate types in validate query requests. {pull}35575[#35575]
+* Deprecate types in count and msearch. {pull}35421[#35421] (issue: {issue}34041[#34041])
+
+Migration::
+* Deprecate X-Pack centric Migration endpoints {pull}35976[#35976] (issue: {issue}35958[#35958])
+
+Monitoring::
+* Deprecate /_xpack/monitoring/* in favor of /_monitoring/* {pull}36130[#36130] (issue: {issue}35958[#35958])
+
+Rollup::
+* Re-deprecate xpack rollup endpoints {pull}36451[#36451] (issue: {issue}36044[#36044])
+* Deprecate X-Pack centric rollup endpoints {pull}35962[#35962] (issue: {issue}35958[#35958])
+
+Scripting::
+* Adds deprecation logging to ScriptDocValues#getValues. {pull}34279[#34279] (issue: {issue}22919[#22919])
+* Conditionally use java time api in scripting {pull}31441[#31441]
+
+Search::
+* Remove X-Pack centric graph endpoints {pull}36010[#36010] (issue: {issue}35958[#35958])
+
+Security::
+* Deprecate X-Pack centric license endpoints {pull}35959[#35959] (issue: {issue}35958[#35958])
+* Deprecate /_xpack/security/* in favor of /_security/* {pull}36293[#36293] (issue: {issue}35958[#35958])
+
+SQL::
+* Deprecate X-Pack SQL translate endpoint {pull}36030[#36030]
+* Deprecate X-Pack centric SQL endpoints {pull}35964[#35964] (issue: {issue}35958[#35958])
+
+Watcher::
+* Deprecate X-Pack centric watcher endpoints {pull}36218[#36218] (issue: {issue}35958[#35958])
+
+
+[[feature-7.0.0-alpha2]]
+[float]
+=== New features
+
+Analysis::
+* Add support for inlined user dictionary in Nori {pull}36123[#36123] (issue: {issue}35842[#35842])
+* Add a prebuilt ICU Analyzer {pull}34958[#34958] (issue: {issue}34285[#34285])
+
+Java High Level REST Client::
+* Add rollup search {pull}36334[#36334] (issue: {issue}29827[#29827])
+
+Java Low Level REST Client::
+* Make warning behavior pluggable per request {pull}36345[#36345]
+* Add PreferHasAttributeNodeSelector {pull}36005[#36005]
+
+Geo::
+* Integrate Lucene's LatLonShape (BKD Backed GeoShapes) as default `geo_shape` indexing approach {pull}36751[#36751] (issue: {issue}35320[#35320])
+* Integrate Lucene's LatLonShape (BKD Backed GeoShapes) as default `geo_shape` indexing approach {pull}35320[#35320] (issue: {issue}32039[#32039])
+
+Machine Learning::
+* [ML-DataFrame] move common fields and string into a utility class {pull}36557[#36557]
```

Comment:
@hendrikmuhs Do we want all these dataframe-related PRs in the release notes?

Reply:
no, it's not in, it's on a feature branch. Sorry for the mess. I will remove the labels on all those PR's

TL;DR;

We want to ship this feature in 7.0 if everything works out, having that said I think these individual changes are to much detail for release notes. Once we approach branch landing we should revisit how we announce it in release notes

Thanks for bringing it up @lcawl and again, sorry for the mess

Reply:
Thanks @hendrikmuhs I removed all those dataframe PRs from this list